src/context/data.tsx

```diff
@@ -181,8 +149,8 @@ export const DataProvider: React.FC<React.PropsWithChildren> = ({
           ...cloneDeep(
             allInventory.features.filter(
               (f) =>
-                f.properties.radius > filterValues.radius.lower / 100 &&
-                f.properties.radius < filterValues.radius.upper / 100 &&
```

Comment:
The conversion now happens only for visual reasons, directly in the range slider component